### PR TITLE
Global zoom variable

### PR DESCRIPTION
**Pull request by dnenov**
### Purpose

**Changes to previous PR "Global zoom variable"**

- removed ZoomLevel from Recourses.resx
- added ZoomLevel to Configurations.cs

Re: This is a minimal quality of life addition - created a resource to contain the value for zoom in/out variable. This value can now be set in one place to affect all converters using it. 

To change the global zoom level variable, go to the Resources of DynamoCoreWpf and change the value of the 'ZoomLevel' variable. At the time of creating this change, the agreed level is **0.4**.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Moved Global Zoom Level Variable to configurations.cs

### Reviewers

@reddyashish 

### FYIs
@Amoursol

### CODE REVIEW

**Comment by Amoursol** [comment]
@dnenov does this help us unblock localization issues such as `0.4` vs `0,4`?

**Comment by QilongTang** [comment]
> @dnenov does this help us unblock localization issues such as `0.4` vs `0,4`?

Yes, this is the proper replacement which will fix it

File: src/DynamoCore/Configuration/Configurations.cs [hunk 1]
```diff
@@ -116,6 +116,11 @@ public class Configurations
         /// </summary>
         public static readonly double ZoomIncrement = 0.05;
 
+        /// <summary>
+        /// Zoom Threshold Level
```
Comment (QilongTang) [inline src/DynamoCore/Configuration/Configurations.cs:120]: Would you be more specific here, e.g. `Global zoom threshold level for node overlay to apply`

File: src/DynamoCore/Configuration/Configurations.cs [hunk 2]
```diff
@@ -116,6 +116,11 @@ public class Configurations
         /// </summary>
         public static readonly double ZoomIncrement = 0.05;
 
+        /// <summary>
+        /// Zoom Threshold Level 
+        /// </summary>
+        public static readonly double ZoomLevel = 0.4;
```
Comment (QilongTang) [inline src/DynamoCore/Configuration/Configurations.cs:122]: Maybe use `ZoomThreshold` as it is clearer just like `ZoomIncrement` above?

**Review by QilongTang — COMMENTED** [review]
Two questions then LGTM

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 3]
```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.15.0.4596")]
```
Comment (reddyashish) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:48]: Revert the changes to this file.

File: src/DynamoCore/Configuration/Configurations.cs (hunk 2 above)
Comment (dnenov) [reply src/DynamoCore/Configuration/Configurations.cs]: Thanks, Aaron! Naming is hard ^^

File: src/DynamoCore/Configuration/Configurations.cs (hunk 1 above)
Comment (dnenov) [reply src/DynamoCore/Configuration/Configurations.cs]: Yup, done! Thank you

**Review by QilongTang — APPROVED** [review]
